ra-front/src/app/services/barrier.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from "@angular/core";
+import { Observable } from "rxjs";
+import { HttpClient } from "@angular/common/http";
+import { StoreService } from "./store.service";
+import { UniversalPoint } from "../model/MapAreas/UniversalPoint";
+import { Polygon } from "../model/MapAreas/Polygons/Polygon";
+
+@Injectable({
+  providedIn: "root",
+})
+export class BarrierService {
+  private readonly barrierURL: string;
+
+  constructor(private http: HttpClient, private store: StoreService) {
+    this.barrierURL = store.barrierURL;
+  }
+
+  public getBarriers(robot_diameter: string): Observable<any> {
+    return this.http.get(
+      this.barrierURL +
+        `dummy_map?map_file_name=${this.store.mapID}&map_folder_name=${this.store.mapID}&robot_diameter=${robot_diameter}`,
```

Comment:
Remove `dummy_map` from this path to use MINIO. Map ID for minio is taken from store.